modules/ui/tools.js

```diff
@@ -15,6 +15,15 @@ import selectBbox       from '../Hoot/tools/selectBbox';
 import { tooltip }      from '../util/tooltip';
 
 export function uiTools( context ) {
+    let isAdvancedUser = false;
+    if ( localStorage ) {
+        const user = JSON.parse( localStorage.getItem( 'user' ) );
+
+        if (user && user.privileges) {
+            isAdvancedUser = user.privileges.advanced === 'true';
```

Comment:
I think the user object in local storage may encode the advanced value as a boolean.  We should double-check this.

Reply:
answered above